### ISSUE TITLE
Checkboxes: Options via API // not using query results in error

### ISSUE THREAD

**Issue by eyekonspi** [k-next/kirby#4985]
## Description
(after asking in the forum, texnixe asked me to post it here as well https://forum.getkirby.com/t/checkboxes-options-via-api-not-using-query-results-in-error/27448)

As per https://getkirby.com/docs/reference/panel/fields/checkboxes#options-via-api
it should be possible, to only have `type: api` & `api: _url_` defined.
But the error I get is:
```
Kirby\Query\Query::factory(): Argument #1 ($query) must be of type string, null given, called in /(..)/kirby/src/Option/OptionsApi.php on line 113 in file: /(..)/kirby/src/Query/Query.php line: 54
```

If I add an empty `query:` to the blueprint, everything works:

```
options:
  type: api
  url: https://my-api.com/bla-api/tags.json
  query:
```

the JSON looks like this:
```
{
  "100% natural": "100% natural (1)",
  "Agility Interview": "Agility Interview (1)",
  (...)
}
```

**Expected behavior**  
It should work without having a `query:` in the blueprint

## Your setup
3.8.3

**Comment by guidoferreyra** [comment]
Happening too on v3.8.2

**Comment by bastianallgeier** [comment]
✅  Will be fixed in 3.9
